discovery-provider/plugins/notifications/src/sns.ts

```diff
@@ -108,6 +108,55 @@ export const sendAndroidMessage = async ({
   })
 }
 
+const formatPushMessage = ({
+  device,
+  title,
+  body,
+  targetARN,
+  data = {},
+  playSound = true
+}: {
+  device: Device
+  title: string
+  body: string
+  targetARN: string
+  data: object
+  playSound: boolean
+}): string => {
+  if (device.type == "android") {
+    return JSON.stringify({
+      default: body,
+      GCM: {
+        notification: {
```

Comment:
one more fix! isaac debugged why android wasn't working, turns out we hvae to stringify here as well (https://github.com/AudiusProject/audius-protocol/commit/1c1de216dfa5e35b9d2bdecbe85035e10ec157bb) mind updating this too?

Reply:
actually wait where does this function get used?

Reply:
this gets used whenever the plugin sends an android push, i'll take a look at Isaacs fix and move it here if needed